data/abilities.js

```diff
@@ -40,6 +40,13 @@ Ratings and how they work:
 
 /**@type {{[k: string]: AbilityData}} */
 let BattleAbilities = {
+	"noability": {
+		shortDesc: "Does nothing.",
+		id: "noability",
+		name: "No Ability",
+		rating: 0.1,
+		num: 0,
```

Comment:
This ability needs the ``isNonstandard`` flag

Reply:
It’s an actual ability though

Reply:
> do we expect this to be released eventually

The answer for that is no. So, it's not unreleased! It is, however, homologue to Missingno as an ability.